test/index.js

```diff
@@ -1,16 +1,16 @@
 const { expect } = require('chai');
```

Comment:
Should test files be `.ts` ?

Reply:
They could be, but arguably they will always be relatively simple scripts so benefit less from strong typing.

Reply:
OK good point.